utam-core/src/main/java/utam/core/framework/context/MobilePlatformType.java

```diff
@@ -86,4 +86,25 @@ public String getConfigName(String moduleName) {
     return getProfileConfigName(this, moduleName);
   }
 
+  private static boolean isiPad(WebDriver driver) {
+    String device =  ((AppiumDriver) driver).getSessionDetail("device").toString();
+    if (device != null && !device.isEmpty()) {
+      return ((AppiumDriver)driver).getSessionDetail("device").toString().equalsIgnoreCase("iPad");
+    }
+    return false;
+  }
+
+  private static boolean isTablet(WebDriver driver) {
+    String[] deviceScreenSize = ((AppiumDriver)driver).getSessionDetail("deviceScreenSize").toString().split("[xX]");
+    String deviceScreenDensity = ((AppiumDriver)driver).getSessionDetail("deviceScreenDensity").toString();
+
+    // For android, based on https://developer.android.com/training/multiscreen/screensizes
+    // when device's dp is equal or bigger than 600, will be treated as tablet, otherwise will be phone
+    if (deviceScreenSize != null && !deviceScreenSize[0].isEmpty() 
+            && deviceScreenDensity != null && !deviceScreenDensity.isEmpty()) {
```

Comment:
nothing can't be null at this point. lines 98, 99 would throw NPE if it were

Reply:
removed the null checks.